### ISSUE TITLE
CCScrollView on desktop web

### ISSUE THREAD

**Issue by HowardBraham** [cocos/cocos-engine#1078]
CCScrollView is a little broken on desktop web.  If you scroll using the mouse wheel, or with two-finger scroll on a touchpad, the result is both backwards and really slow.

For example, if Google Chrome would normally respond to the mouse wheel gesture by scrolling down about 40 pixels, a Cocos ScrollView will respond by scrolling **up** about 2 pixels.

I believe this can be fixed by changing CCScrollView.js line 702 from
`var wheelPrecision = 1.0 / 40;`
to
`var wheelPrecision = -0.6;`

Negative 0.6 feels about right, but see what you think.

**Comment by zilongshanren** [comment]
@HowardBraham  Thanks, I will try the solution and give you feedback.

**Comment by zilongshanren** [comment]
@HowardBraham 
`-0.6` might be too aggressive, so I change it to `-0.1`

And here is the PR: https://github.com/cocos-creator/engine/pull/1095

Feel free to comment, thanks.

**Comment by HowardBraham** [comment]
@zilongshanren 
It looks like this change was merged, but didn't make it into Cocos Creator 1.2.2.  What happened?

Also, in terms of the best wheelPrecision, I've studied this a bit more and I think it's more complicated.  Here are two test pages I set up with wheelPrecisions of -0.1 and -0.6:

https://howardbraham.github.io/0.1/
https://howardbraham.github.io/0.6/

On the left is a Cocos scroll, and on the right is a native HTML scroll.  To me, on Windows Firefox, the value seems right at -0.1.  But on Windows Chrome, it seems right at -0.6.

Cocos on Firefox is also affected by my Windows setting called "Choose how many lines to scroll each time," while Cocos on Chrome is not affected by this.

I have not tested on Mac or Linux.  But I think we may need to keep working on this in order to make a consistent experience across all platforms.
